reactor-core/src/test/java/reactor/core/publisher/FluxReplayTest.java

```diff
@@ -572,7 +650,7 @@ public void ifNoSubscriptionBeforeConnectThenUnbounded() {
 		            .expectComplete()
 		            .verify(Duration.ofSeconds(10));
 
-		assertThat(totalRequested).hasValue(Long.MAX_VALUE);
+		assertThat(totalRequested).hasValue(12L);
```

Comment:
rename the test, since it doesn't assert an unbounded initial request anymore

Reply:
fixed